packages/calypso-doctor/cli.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+const yargs = require( 'yargs' );
+const chalk = require( 'chalk' );
+const { runEvaluations } = require( './index.js' );
+
+const main = async ( args ) => {
+	const results = await runEvaluations( { fix: args.fix } );
```

Comment:
The argument to `runEvaluations` is unused. Is that intentional or left over from a previous idea?

Reply:
Left over from my attempt to implement autofix, I'll clean it up, thanks!